Sources/ConsoleKit/Utilities/ConsoleLogger.swift

```diff
@@ -1,36 +1,95 @@
-#warning("TODO: update to SSWG logging library")
-//import Logging
-//
-///// Outputs logs to a `Console`.
-//public final class ConsoleLogger: Logger {
-//    /// The `Console` logs will be outputted to.
-//    public let console: Console
-//
-//    /// Create a new `ConsoleLogger`.
-//    public init(console: Console) {
-//        self.console = console
-//    }
-//
-//    /// See `Logger`.
-//    public func log(_ string: String, at level: LogLevel, file: String, function: String, line: UInt, column: UInt) {
-//        let text: ConsoleText = ""
-//            + "[ \(level) ]".consoleText(level.style)
-//            + " "
-//            + string.consoleText()
-//            + " "
-//            + "(\(file):\(line))".consoleText(.info)
-//        console.output(text)
-//    }
-//}
-//
-//extension LogLevel {
-//    /// Converts log level to console style
-//    fileprivate var style: ConsoleStyle {
-//        switch self {
-//        case .custom, .verbose, .debug: return .plain
-//        case .error, .fatal: return .error
-//        case .info: return .info
-//        case .warning: return .warning
-//        }
-//    }
-//}
+import Logging
+
+/// Outputs logs to a `Console`.
+public struct ConsoleLogger: LogHandler {
+    
+    /// See `LogHandler.metadata`.
+    public var metadata: Logger.Metadata
+    
+    /// See `LogHandler.logLevel`.
+    public var logLevel: Logger.Level
+    
+    /// The conosle that the messages will get logged to.
+    public let console: Console
+    
+    /// Creates a new `ConsoleLogger` instance.
+    ///
+    /// - Parameters:
+    ///   - console: The console to log the messages to.
+    ///   - level: The minimum level of message that the logger will output. This defaults to `.debug`, the lowest level.
+    ///   - metadata: Extra metadata to log with the message. This defaults to an empty dictionary.
+    public init(console: Console, level: Logger.Level = .debug, metadata: Logger.Metadata = [:]) {
+        self.metadata = metadata
+        self.logLevel = level
+        self.console = console
+    }
+    
+    /// Creates a boottrap function that will create a `ConsoleLogger` instance.
+    ///
+    ///     LoggingSystem.boostrap(ConsoleLogger.bootstrap(console: console))
+    ///
+    /// - Parameters:
+    ///   - console: The console the logger will log the messages to.
+    ///   - level: The minimum level of message that the logger will output. This defaults to `.debug`, the lowest level.
+    ///   - metadata: Extra metadata to log with the message. This defaults to an empty dictionary.
+    ///
+    /// - Returns: A function that can be passed into the `LoggingSystem.bootstrap(_:)` method.
+    public static func bootstrap(
```

Comment:
Maybe we could instead provide this as an extension on `LoggingSystem`? The `-> (String) -> LogHandler` return seems a little weird, but maybe that's just me. 

```swift
import Logging
import ConsoleKit

LoggingSystem.bootstrap(console: myConsole)
```

Maybe this could be nice? I'd be interested what kind of pattern @weissi had in mind.

Reply:
I like that 👍. Either you can add the change or I'll do it when I get a chance.